cmd/kubetoken/password.go

```diff
@@ -36,7 +34,12 @@ func getPassword(user string, skipKeyringFetch bool) string {
 	// Password was not found, prompt the user for it
 	// Save the password in the keyring
 	password = promptForPassword(user)
-	check(setKeyringPassword(user, password))
+
+	err := setKeyringPassword(user, password)
```

Comment:
If we were to change it to do this, there would be no way to override or reset the currently stored password in the keyring. If the user was to use kubetoken without the `--password-prompt` option, it would revert back to using an old password from the keyring.

I envisioned the usage of `--password-prompt` as a way to update the currently stored password with a new password.

Reply:
I would make a separated flag to update the password in keyring. This works if you want to use it update the stored password while the only drawback is that it will always print the warning message if you don't have a keyring.